### PR TITLE
COMMERCE-5411 Remove portlet 90 configuration for BOM

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

File: modules/dxp/apps/commerce/commerce-bom-service/src/main/resources/resource-actions/default.xml [hunk 1]
```diff
@@ -2,8 +2,14 @@
 <!DOCTYPE resource-action-mapping PUBLIC "-//Liferay//DTD Resource Action Mapping 7.0.0//EN" "http://www.liferay.com/dtd/liferay-resource-action-mapping_7_0_0.dtd">
 
 <resource-action-mapping>
-	<portlet-resource>
-		<portlet-name>90</portlet-name>
+	<model-resource>
+		<model-name>com.liferay.commerce.bom</model-name>
+		<portlet-ref>
+			<portlet-name>com_liferay_commerce_bom_admin_web_internal_portlet_CommerceBOMAdminPortlet</portlet-name>
+			<portlet-name>com_liferay_commerce_bom_admin_web_internal_portlet_CommerceBOMAdminPortlet</portlet-name>
```
Comment (dsitu) [inline modules/dxp/apps/commerce/commerce-bom-service/src/main/resources/resource-actions/default.xml:9]: @riccardo-alberti duplicate?

**Review by dsitu — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/commerce/commerce-bom-service/src/main/resources/resource-actions/default.xml (hunk 1 above)
Comment (riccardo-alberti) [reply modules/dxp/apps/commerce/commerce-bom-service/src/main/resources/resource-actions/default.xml]: thanks!

**Comment by riccardo-alberti** [comment]
ci:forward

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**